src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -264,25 +268,68 @@ const Packages = ({ getAllPackages, isSuccess }) => {
     setFilterChosen('');
   };
 
+  const ExactMatch = ({ pkgList, search }) => {
```

Comment:
This actually seems to be inside of the <Packages> component. It works, but we don't have nested components anywhere else in the codebase, so maybe for the sake of consistency we should move it outside of <Packages>? (It can stay in Packages.js, of course...)

Reply:
You're right, I've moved the `ExactMatch` outside of the `Packages` component :+1: